VERSION

```diff
@@ -1 +1 @@
-0.9.0
+1.0.0
```

Comment:
Let the master be 1.1.0 :-)

Reply:
The PR to 1.0.0 -- can be directly raised on the v1.0.x branch.

Reply:
Done